tests/unit/requirements.txt

```diff
@@ -12,3 +12,7 @@ pyFMG
 
 # requirements for routeros_api module
 librouteros ; python_version >= '3.6'
+
+# requirements for eos_pyateos modules
+jsondiff
```

Comment:
Not sure why you added these requirements, since you didn't add unit tests?

Reply:
If you only need these requirements in integration tests, you need to install them in the integration tests (use the `pip` module).

Reply:
That is a requirement for making the module to work. I can see other modules putting specific requirements under `DOCUMENTATION` as well, so I did the same.